templates/design_system/base.html

```diff
@@ -27,5 +27,9 @@
 		{% if template %}{% include template %}{% endif %}
 		{{ content|safe }}
 	{% endblock %}
+
+
+    <script>window.tealiumEnvironment = "{{ settings.GOOGLE_AN_ENV }}";</script>
```

Comment:
I'm not seeing GOOGLE_AN_ENV in the settings file updates, is it being set elsewhere?